daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Speedy.scala

```diff
@@ -166,8 +166,8 @@ object Speedy {
               CtrlExpr(body)
             case None =>
               throw SpeedyHungry(
-                SResultMissingDefinition(
-                  ref, { packages =>
+                SResultMissingPackage(
```

Comment:
It seems like it might be nice to differentiate between “this package is not in `compiledPackages`” which can happen legitimately and “the package is there but the definition is not” which is a fatal error afaict.

Reply:
With my change, I think the two are now conceptually different. 
* `MissingDefinition` is an error  
* `MissingPackage` is a Package request to the interpret (Note I change the name to `NeedPackage`